apps/src/music/ai/patternAiWorker.ts

```diff
@@ -0,0 +1,167 @@
+// Carefully import only what works in a web worker.
+import mm from '@magenta/music/es6';
+import {sequences} from '@magenta/music/es6/core';
+import {MusicRNN} from '@magenta/music/es6/music_rnn';
```

Comment:
It took a while to figure out this minimal set of imports that allowed the generation to run in a web worker.